foundation/oidc/src/main/kotlin/com/pingidentity/oidc/OidcWeb.kt

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2025 Ping Identity Corporation. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.pingidentity.oidc
+
+import com.pingidentity.oidc.module.OidcFlow
+import com.pingidentity.oidc.module.PARAMETERS
+import com.pingidentity.oidc.module.oidcClientConfig
+import com.pingidentity.oidc.module.oidcUser
+import com.pingidentity.oidc.module.prepareUser
+import com.pingidentity.oidc.module.user
+import com.pingidentity.oidc.module.Web
+import com.pingidentity.orchestrate.FailureNode
+import com.pingidentity.orchestrate.SharedContext
+import com.pingidentity.orchestrate.SuccessNode
+import com.pingidentity.orchestrate.WorkflowConfig
+
+/**
+ * OIDC Web configuration class.
+ * Provide configuration for the OIDC web module.
+ */
+class OidcWebConfig : WorkflowConfig() {
+    // additional configuration for example browser settings
```

Comment:
Don't you have any browser configuration properties to add here?

Reply:
Will consider this use case when work on the AuthTab Story.